.github/workflows/bot-phosphore.yml

```diff
@@ -10,6 +10,7 @@ on:
     branches:
       - support/bot-phosphore
       - support/bot-phosphore-*
+      - feat/LIVE-2572-POC-modularisation-coin-families
```

Comment:
why adding this ?

Reply:
Removed, it was the old way of testing with the bot i think